### PR TITLE
refactor(bridge): new influxDB (WIP)

### PR DESCRIPTION
**Pull request by Kinplemelon**

### CODE REVIEW

**Comment by Kinplemelon** [comment]
merge https://github.com/emqx/emqx-dashboard-web-new/pull/505 first, so the CI in this PR can pass

**Comment by ysfscream** [comment]
CI Failed @Kinplemelon

File: src/i18n/RuleEngine.js [hunk 1]
```diff
@@ -871,4 +855,104 @@ export default {
     zh: '将数据保存到 MySQL',
     en: 'Save data to MySQL',
   },
+  influxDBVersion: {
+    zh: 'InfluxDB 版本',
+    en: 'Version of InfluxDB',
```
Comment (ysfscream) [inline src/i18n/RuleEngine.js:860]: Version

File: src/i18n/RuleEngine.js [hunk 2]
```diff
@@ -839,29 +839,13 @@ export default {
     zh: 'messages/sec | message/sec | messages/sec',
     en: 'messages/sec | message/sec | messages/sec',
   },
-  influxDBV1Label: {
+  influxDBLabel: {
     zh: 'InfluxDB HTTP 服务',
     en: 'InfluxDB HTTP Service',
```
Comment (ysfscream) [inline src/i18n/RuleEngine.js:844]: InfluxDB

without `HTTP Service`

File: src/i18n/RuleEngine.js [hunk 3]
```diff
@@ -839,29 +839,13 @@ export default {
...
-  },
-  influxDBUDPLabel: {
-    zh: 'InfluxDB UDP 服务',
-    en: 'InfluxDB UDP Service',
-  },
-  influxDBUDPDesc: {
-    zh: '使用 UDP 协议将数据保存到 InfluxDB',
-    en: 'Save data to InfluxDB using UDP protocol',
+  influxDBDesc: {
+    zh: '使用 HTTP 协议将数据保存到 InfluxDB',
```
Comment (ysfscream) [inline src/i18n/RuleEngine.js:847]:
```suggestion
    zh: '将数据保存到 InfluxDB',
```

File: src/i18n/RuleEngine.js [hunk 4]
```diff
@@ -839,29 +839,13 @@ export default {
...
+    en: 'Save data to InfluxDB using HTTP protocol',
```
Comment (ysfscream) [inline src/i18n/RuleEngine.js:848]:
```suggestion
    en: 'Save data to InfluxDB',
```

File: src/i18n/RuleEngine.js [hunk 5]
```diff
@@ -871,4 +855,104 @@ export default {
...
+    en: 'The IPv4 or IPv6 address or the hostname to connect to.</br>A host entry has the following form: `Host[:Port]`.</br>The InfluxDB default port 8086 is used if `[:Port]` is not specified.</br>',
+    zh: '将要连接的 IPv4 或 IPv6 地址，或者主机名。</br>主机名具有以下形式：`Host[:Port]`。</br>如果未指定 `[:Port]`，则使用 InfluxDB 默认端口 8086。</br>',
+  },
+  database: {
+    en: 'Database',
+    zh: '数据库',
+  },
+  databaseDesc: {
+    en: 'InfluxDB database.',
+    zh: 'InfluxDB 数据库。',
```
Comment (ysfscream) [inline src/i18n/RuleEngine.js:876]: Remove InfluxDB

File: src/i18n/RuleEngine.js [hunk 6]
```diff
@@ -871,4 +855,104 @@ export default {
...
+  },
+  precision: {
+    en: 'Time Precision',
+    zh: '时间精度',
+  },
+  precisionDesc: {
+    en: 'InfluxDB time precision.',
+    zh: 'InfluxDB 时间精度。',
```
Comment (ysfscream) [inline src/i18n/RuleEngine.js:884]: Remove InfluxDB

File: src/i18n/RuleEngine.js [hunk 7]
```diff
@@ -871,4 +855,104 @@ export default {
...
+    zh: '自动重连间隔',
+  },
+  autoRestartIntervalDesc: {
+    en: 'The auto restart interval after the resource is disconnected, in milliseconds.',
+    zh: '资源断开以后，自动重连的时间间隔，单位毫秒。',
+  },
+  token: {
+    en: 'Token',
+    zh: 'Token',
+  },
```
Comment (ysfscream) [inline src/i18n/RuleEngine.js:921]: 都是英文就不需要翻译了？

**Review by ysfscream — APPROVED** [review]

**Pull request merged**